e2e_tests/integration/auto-prefix.spec.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2002-2021 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+describe(':auto prefix in browser', () => {
+  before(() => {
+    cy.visit(Cypress.config('url')).title().should('include', 'Neo4j Browser')
+    cy.wait(3000)
+    cy.ensureConnection()
+  })
+  beforeEach(() => {
+    cy.executeCommand(':clear')
+  })
+
+  it('shows help link when running period commit without :auto', () => {
+    cy.executeCommand('USING PERIODIC COMMIT RETURN "Verdanturf"')
+    cy.getFrames().contains('ERROR')
+    cy.getFrames().contains(':auto')
+  })
+
+  it('adding :auto enables running periodic commit', () => {
+    cy.executeCommand(':auto USING PERIODIC COMMIT RETURN "Laverre";')
+    cy.getFrames().contains('ERROR')
```

Comment:
Could you add a comment explaining why we're looking for an error here?

Reply:
sure, it definitely looks a bit odd